### PR TITLE
chore: setup sentry for the project

### LINKED ISSUE #14: Setup sentry
https://github.com/vercel/next.js/tree/canary/examples/with-sentry

### PR DESCRIPTION
**Pull request by pc-beast**
Fixes #14

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Remove all test pages and api

**Comment by pc-beast** [comment]
done

File: tsconfig.json [hunk 1]
```diff
@@ -14,6 +14,23 @@
...
+    "pages/ssr/test1.js",
+    "pages/ssr/test2.js",
+    "pages/client/test5.js",
+    "pages/client/test4.js",
+    "pages/sentry.js",
+    "pages/sentry.js",
+    "pages/api/test4.js",
+    "pages/client/test1.js",
+    "pages/client/test2.js",
+    "pages/client/test3.js"
```
Comment (iamareebjamal) [inline tsconfig.json:33]: Remaining, and there was absolutely no need to put them here at all

Comment (pc-beast) [reply tsconfig.json]: These were put by my linter automatically ig. I am really sorry for it.

File: utils/sentry.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import * as Sentry from '@sentry/node'
+import { RewriteFrames } from '@sentry/integrations'
+
+export const init = (): void => {
```
Comment (iamareebjamal) [inline utils/sentry.ts:4]: This is not called anywhere

**Pull request merged**